packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,511 @@
+        codeIndex++;
+        textIndex += codeBlock.text.length;
+      }
+    }
+    if (textIndex < text.length) {
+      eltChildren.push(<span key={textIndex}>{text.substring(textIndex, text.length)}</span>);
+    }
+
+    return eltChildren;
+  }
+
+  private _renderClass(): JSX.Element | undefined {
+    const { properties, methods } = this.state;
+
+    return methods ? (
+      <Stack gap={MEDIUM_GAP_SIZE}>
+        <Stack gap={SMALL_GAP_SIZE}>
+          <Text variant={'medium'}>Members</Text>
+          <DetailsList
+            selectionMode={SelectionMode.none}
+            layoutMode={DetailsListLayoutMode.justified}
+            items={properties}
+            columns={this._defaultColumns}
+            onRenderRow={this._onRenderRow}
+          />
+        </Stack>
+        <Stack gap={SMALL_GAP_SIZE}>
+          <Text variant={'medium'}>Methods</Text>
+          <DetailsList
+            selectionMode={SelectionMode.none}
+            layoutMode={DetailsListLayoutMode.justified}
+            items={methods}
+            columns={this._methodColumns}
+            onRenderRow={this._onRenderRow}
+          />
+        </Stack>
+      </Stack>
+    ) : (
+      <Stack gap={SMALL_GAP_SIZE}>
+        <Text variant={'medium'}>Members</Text>
+        <DetailsList
+          selectionMode={SelectionMode.none}
+          layoutMode={DetailsListLayoutMode.justified}
+          items={properties}
+          columns={this._defaultColumns}
+          onRenderRow={this._onRenderRow}
+        />
+      </Stack>
+    );
+  }
+
+  private _renderExtends(): JSX.Element | undefined {
+    const { extendsTokens } = this.props;
+
+    return this._parseILinkTokens(true, extendsTokens);
+  }
+
+  private _renderDescription(): JSX.Element | undefined {
+    const { description } = this.props;
+
+    return description ? <Text variant={'medium'}>{description}</Text> : undefined;
+  }
+
+  private _renderTitle(): JSX.Element | undefined {
+    const { title, name } = this.props;
+
+    return title ? (
+      <Text variant={'xLarge'} id={name}>
+        {title}
+      </Text>
+    ) : (
+      undefined
+    );
+  }
+
+  private _onRenderRow = (props: IDetailsRowProps, defaultRender?: IRenderFunction<IDetailsRowProps>): JSX.Element => {
+    const { item } = props;
+    const rowStyles: Partial<IDetailsRowStyles> = {
+      root: {
+        selectors: {
+          ':hover': {
+            background: 'transparent'
+          }
+        }
+      },
+      isMultiline: {
+        wordBreak: 'break-word'
+      }
+    };
+
+    if (item.deprecated === true) {
+      const deprecatedStyles: Partial<IDetailsRowStyles> = {
+        root: {
+          background: DEPRECATED_ROW_COLOR,
+          selectors: {
+            ':hover': {
+              background: DEPRECATED_ROW_COLOR
+            }
+          }
+        }
+      };
+
+      return <DetailsRow {...props} styles={rowStyles && deprecatedStyles} />;
```

Comment:
```suggestion
      return <DetailsRow {...props} styles={{ ...rowStyles, ...deprecatedStyles }} />;
```

Assuming you want to merge the styles I think the above would be more clear.

**OR** you can probably just do this merge _once_ and use it in this conditional block.